components/dot_menu/dot_menu.jsx

```diff
@@ -219,6 +226,24 @@ export default class DotMenu extends Component {
                 return item.filter ? item.filter(this.props.post.id) : item;
             }).
             map((item) => {
+                if (item.text.subMenu) {
+                    return (
+                        <SubMenuItemAction
+                            key={item.id + '_pluginmenuitem'}
+                            id={item.text.id}
+                            text={item.text.text}
+                            subMenu={item.text.subMenu}
+                            action={(id) => {
+                                if (item.action) {
+                                    item.action(this.props.post.id, id);
+                                }
+                            }}
+                            openModal={this.props.actions.openModal}
```

Comment:
Is there any benefit to passing this down as a prop? I believe this can be mapped directly into the submenu component.

Reply:
Great catch!  `openModal` is now directly imported.